### PR TITLE
Extract file variable

### PR DESCRIPTION
**Pull request by RoiEXLab**
Follow-up to #6002

## Functional Changes

[] New map or map update
[] New Feature
[] Feature update or enhancement
[] Feature Removal
[x] Code Cleanup or refactor
[] Configuration Change
[] Problem fix:  
[] Other:

### CODE REVIEW

**Comment by DanVanAtta** [comment]
@RoiEXLab any thoughts to take care of the code climate warning?

File: game-core/src/main/java/games/strategy/engine/framework/GameRunner.java [hunk 1]
```diff
@@ -90,19 +91,20 @@ public static void showMainFrame() {
     }
   }
 
-  private static void loadGame() {
+  private static void loadGameIfSpecified() {
     checkState(!SwingUtilities.isEventDispatchThread());
     gameSelectorModel.loadDefaultGameSameThread();
-    final String fileName = System.getProperty(TRIPLEA_GAME, "");
-    if (!fileName.isEmpty() && new File(fileName).exists()) {
+    final File saveGameFile = new File(System.getProperty(TRIPLEA_GAME, ""));
```
Comment (RoiEXLab) [inline game-core/src/main/java/games/strategy/engine/framework/GameRunner.java]: Note: The `File` constructor doesn't throw an exception when using an empty path and seems to return false for `new File("").exists()`, so the empty check is redundant.

This way the cognitive complexity should go down.

**Comment by RoiEXLab** [comment]
@DanVanAtta Updated, tell me what you think

File: game-core/src/main/java/games/strategy/engine/framework/GameRunner.java (hunk 1 above)
Comment (DanVanAtta) [reply game-core/src/main/java/games/strategy/engine/framework/GameRunner.java]: That's a good simplification :+1:

**Pull request merged**